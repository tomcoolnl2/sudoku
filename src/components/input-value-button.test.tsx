
import { render } from '@testing-library/react'
import { withTheme, withRedux } from '../test'
import { InputValueButton, InputValueProps } from './'


const props: InputValueProps = {
	value: 1
}

describe('InputValueButton', () => {
	
	it('should match it\'s snapshot', () => {
		const { container } = render(withTheme(withRedux(<InputValueButton {...props} />)))
		expect(container.firstChild).toMatchSnapshot()
	})
})