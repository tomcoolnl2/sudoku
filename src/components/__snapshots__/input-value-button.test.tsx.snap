// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputValueButton should match it's snapshot 1`] = `
<button
  class="sc-gKseQn hFABYf"
>
  1
</button>
`;
